debian/po/pt_BR.po

```diff
@@ -2350,9 +2347,8 @@ msgid "FIPS Updates"
 msgstr ""
 
 #: ../../uaclient/messages/__init__.py:1320
-#, fuzzy
 msgid "FIPS compliant crypto packages with stable security updates"
-msgstr "Pacotes instalados com atualizações disponíveis por {service}:"
+msgstr ""
 
```

Comment:
You removed the incorrect translation, but didn't add a fixed one?

Reply:
The reason for that is because we have not translated any service-related messages for this release. So for consistency sake, I left that one untranslated too